sys/linux/filesystem.txt

```diff
@@ -86,65 +86,59 @@ loop_filename {
 	z	const[0, int8]
 } [packed]
 
-syz_read_part_table(size intptr, nsegs len[segments], segments ptr[in, array[fs_image_segment]])
+syz_read_part_table(size intptr, size_compressed len[img], img ptr[in, compressed_image]) (no_generate, no_minimize)
```

Comment:
We now don't generate this syscall and we also don't have seeds.
Please file an issue to add seeds for this syscall, or to check if we need to switch to a normal `array[int8]` for the input argument. Not sure how large the inputs should be to get good coverage for this syscall, but the fuzzer managed to trigger a number of bugs with this syscall even w/o seeds.

Reply:
Filed as #3529